libs/widget-lib/src/types.ts

```diff
@@ -85,7 +85,7 @@ interface CowSwapWidgetConfig {
   height: string
   /**
    * The unique identifier of the widget consumer.
-   * Please fill the for to let us know a little about you: <TODO-TYPEFORM>
+   * Please fill the for to let us know a little about you: https://cowprotocol.typeform.com/to/rONXaxHV
```

Comment:
https://github.com/cowprotocol/cowswap/pull/3351

Reply:
those changes were not meant to be there, i used the wrong branch